src/config/init.js

```diff
@@ -0,0 +1,35 @@
+import bcrypt from 'bcrypt';
+import config from './config';
+import models from '../database/models';
+import UserService from '../services/user.service';
+
+const { User } = models;
+const { createUser } = UserService;
+
+const adminScript = async () => {
+  const salt = await bcrypt.genSalt(10);
+  const hashedPassword = await bcrypt.hash(config.adminAccountPassword, salt);
+
+  const admin = await User.findOne({
+    where: { email: config.adminAccountEmail },
+  });
+
+  if (!admin) {
+    createUser({
+      firstName: config.adminAccountFirstname,
+      lastName: config.adminAccountLastname,
+      email: config.adminAccountEmail,
+      userName: config.adminAccountUsername,
+      password: hashedPassword,
+      roleId: config.adminAccountRoleId,
```

Comment:
Consider the **D**on't **R**epeat **Y**ourself approach

Reply:
@niyojuste Check the flow please I am not Repeating myself.